src/rust/ide/src/controller/visualization.rs

```diff
@@ -0,0 +1,254 @@
+//! Visualization controller.
+//!
+//! Ths Visualization Controller is Responsible identifying all the available visualization natively
+//! embedded in IDE and available within the project's `visualization` folder. The
+//! `Visualization Controller` lives as long as the `Project Controller`.
```

Comment:
> The `Visualization Controller` lives as long as the `Project Controller`.

I'd say this is more of a Project Controller's property rather than Visualization Controller.

Reply:
I am removing this statement from this module.